<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelLoaderUI</class>
 <widget class="QDialog" name="ModelLoaderUI">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>422</width>
    <height>189</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>422</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModelLoaderUI</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupMesh">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Сетка геометрии .obj - Файл</string>
   </property>
   <widget class="QPushButton" name="btnOpenObjFile">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMeshFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupShader">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Тип шейдера</string>
   </property>
   <widget class="QComboBox" name="cbxShaderType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupTexture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Текстуры .dds</string>
   </property>
   <widget class="QPushButton" name="btnOpenDDSFile">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextureFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTex1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTex1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текстура 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTex2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDDSFile_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTex2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текстура 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTex3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDDSFile_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTex3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текстура 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledTex4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDDSFile_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTex4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текстура 4</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>ModelLoaderUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ModelLoaderUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
